buildSrc/src/main/groovy/scripts/Utils.groovy

```diff
@@ -49,6 +49,8 @@ class Utils {
             'provider': 'gradle',
             'Implementation-Title': title,
             'Implementation-Version': "$version $revision",
+            'Bundle-Version': "$version",
+            'BuildInfo-Timestamp': new Date().format("yyyy-MM-dd'T'HH:mm:ssZ"),
```

Comment:
So this is a little awkward from a Gradle point of view... I'd also argue it's not massively helpful from a diagnostic POV. I would think the current `Implementation-Version` attribute format is a more informative thing that the version + timestamp combination.